subprojects/gradle/lint-report/build.gradle.kts

```diff
@@ -14,6 +14,7 @@ dependencies {
     implementation(libs.okhttp)
     implementation(libs.androidToolsLintApi)
 
+    implementation(project(":common:http-client"))
     implementation(project(":common:okhttp"))
```

Comment:
it server different purposes, 
`:common:http-client` only provides pre-baked httpClientBuilders, and
`:common:okhttp` contains extensions

if you need both - you include both

Reply:
It's not clear what those modules contain
maybe we should rename okhttp to okhttp-extensions

Reply:
sure; but there is a problem with such renaming, because it's used in avito
we can change module name, and not change publication artifact name; but that will add even more confusion :(

Reply:
We could make breaking change with naming